pkg/operator/ceph/controller/spec.go

```diff
@@ -67,6 +67,25 @@ const (
 	startupProbeFailuresDaemonOSD int32 = 12 * 60
 )
 
+// mds probe constants
+const (
+	// set this to be close to the probe period to avoid routine timeouts, also avoids
+	// unlikely routine leaks in the case of some really unexpected failure mode in the probe script
+	mdsTimeoutSeconds int32 = 25
+	// timeout for the mds cmd, should be less than mdsTimeoutSeconds
+	mdsCmdTimeout int32 = 20
+	// set this equal to (or greater than) the probe period so we allow the daemon time
+	// to stabilize a bit before querying it
+	mdsInitialDelaySeconds int32 = 30
+	// we want to balance checking too often with getting the cluster back to healthy in a reasonable time-frame
+	mdsPeriodSeconds int32 = 30
+	// this should also prevent unnecessary MDS restarts when Rook is initially creating
+	// the filesystem and waiting for MDSes to join for the first time
+	mdsFailureThreshold int32 = 5
+	//  we have strong certainty of success, and a low value keeps from restarting the MDS pod too often in flaky systems
+	mdsSuccessThreshold int32 = 1
+)
+
```

Comment:
```suggestion
const (
	// set this to be close to the probe period to avoid routine timeouts, also avoids
	// unlikely routine leaks in the case of some really unexpected failure mode in the probe script
	mdsTimeoutSeconds int32 = 25
	// timeout for the mds cmd, should be less than mdsTimeoutSeconds
	mdsCmdTimeout int32 = 20
	// set this equal to (or greater than) the probe period so we allow the daemon time
	// to stabilize a bit before querying it
	mdsInitialDelaySeconds int32 = 30
	// we want to balance checking too often with getting the cluster back to healthy in a reasonable time-frame
	mdsPeriodSeconds int32 = 30
	// this should also prevent unnecessary MDS restarts when Rook is initially creating
	// the filesystem and waiting for MDSes to join for the first time
	mdsFailureThreshold int32 = 5
	//  we have strong certainty of success, and a low value keeps from restarting the MDS pod too often in flaky systems
	mdsSuccessThreshold int32 = 1
)

```

why these are const var? we can have it as `var()`?

Reply:
As these values have to be constant always.